### DISCUSSION TITLE
TimeField - increment/decrement interval

### DISCUSSION THREAD

**Discussion by garrett-thompson** [adobe/react-spectrum#5716] [Ideas]
Thoughts on making the `increment` and `decrement` methods of `useTimeFieldState` take an optional `interval` param that would override the `1` value here: https://github.com/adobe/react-spectrum/blob/ee51290b98fdc7cad31957ed2dd0a486eb0c1427/packages/%40react-stately/datepicker/src/useDateFieldState.ts#L353-L358?

For instance, I want to increment the minute field by 15 or 30 minutes at a time.

**Comment by yihuiliao** [comment]
There's been a previous discussion with an open ticket about this #4410
